### DISCUSSION TITLE
ZbStatus2 return {"ZbStatus":"Unknown device"}

### DISCUSSION THREAD

**Discussion by bmc-git** [arendst/Tasmota#11520] [Q&A] [answered]
Hi, is there anybody which has been able to get a list of devices with ZbStatus2 without any other argumens. Reading the manual it seems that omitting the device should return all devices.

If I specify a device I will get the status of that device, but omitting the device will give me {"ZbStatus":"Unknown device"}

**Comment by s-hadinger** [comment, marked as answer]
That's on the plan to allow the list of all devices. Currently it's not supported. ZbStatus does give you the list. You can also iterate by numbers: `ZbStatus 1`, `ZbStatus 2`...
